packages/react-aria-components/stories/ListBox.stories.tsx

```diff
@@ -187,3 +190,74 @@ export const ListBoxHover = () => (
   </ListBox>
 );
 
+export function VirtualizedListBox() {
+  let sections: {id: string, name: string, children: {id: string, name: string}[]}[] = [];
+  for (let s = 0; s < 10; s++) {
+    let items: {id: string, name: string}[] = [];
+    for (let i = 0; i < 100; i++) {
+      items.push({id: `item_${s}_${i}`, name: `Item ${i}`});
+    }
+    sections.push({id: `section_${s}`, name: `Section ${s}`, children: items});
+  }
+  return (
+    <VirtualizedCollection>
+      <ListBox className={styles.menu} style={{height: 400}} aria-label="virtualized listbox" items={sections}>
+        {section => (
+          <Section className={styles.group}>
+            <Header style={{fontSize: '1.2em'}}>{section.name}</Header>
+            <Collection items={section.children}>
+              {item => <MyListBoxItem>{item.name}</MyListBoxItem>}
+            </Collection>
+          </Section>
+        )}
+      </ListBox>
+    </VirtualizedCollection>
+  );
+}
+
+function VirtualizedCollection({children}) {
+  return (
+    <CollectionRendererContext.Provider value={VirtualizedCollectionRenderer}>
+      {children}
+    </CollectionRendererContext.Provider>
+  );
+}
+
+const VirtualizedCollectionRenderer: CollectionRenderer = (collection, parent) => {
+  if (parent) {
+    // eslint-disable-next-line react-hooks/rules-of-hooks
+    let virtualizer = useContext(VirtualizerContext)!;
+    return virtualizer.virtualizer.getChildren(parent.key);
+  }
+
+  // eslint-disable-next-line react-hooks/rules-of-hooks
+  let layout = useMemo(() => {
+    return new ListLayout({
+      estimatedRowHeight: 32,
+      estimatedHeadingHeight: 26,
+      padding: 4,
+      loaderHeight: 40,
+      placeholderHeight: 32
+      // collator
+    });
+  }, []);
+
+  return (
+    <Virtualizer
+      sizeToFit="height"
+      scrollDirection="vertical"
+      layout={layout}
+      style={{height: 'inherit'}}
+      collection={collection}
+      shouldUseVirtualFocus>
```

Comment:
I used that so that we don't apply a `tabIndex` to the virtualizer element. In RSP, the virtualizer is the outer most element (the one with `role="listbox"`) so this made sense. But here, the virtualizer is rendered inside the listbox (it has `role="presentation"`), so it should not also have a tabIndex. Really virtualizer is duplicating the logic already handled by the listbox/selection hooks, so it isn't really necessary.